### ISSUE TITLE
DevTools in production

### ISSUE THREAD

**Issue by dbuchet** [jotaijs/jotai-devtools#49]
Hello,
I there any issue with last version of jotai-devtools (`0.3.1`) when building app in production mode?

I can see from the documentation
> All the components and the hooks are optimized to be tree-shakable for production builds and only works in a non-production environment

But when I build & deploy my app, I still can see the Jotai Devtools console button. My stores are all labelled `<unlabelled-atom>` (whereas have proper label in development mode)

I'm also using react-query DevTools, which is well excluded from production (so yes, I'm in production mode :) )

Just wanted to check if i'm doing something wrong, if someone else have this issue, or if I have to add a condition myself to exclude DevTools on production?

**Comment by arjunvegda** [comment]
Are you using CRA to bundle your application by any chance? See: https://github.com/jotaijs/jotai-devtools/pull/44#issuecomment-1474866468

It's likely that your bundler is importing `esm` modules and does not support `import.meta` syntax that Jotai and Jotai DevTools rely on to identify the environment. 

If you're not using CRA and are running into this issue, then I'd like to learn more about your use case and would be awesome if you could provide a small repro. 

Also, come to think of it, the wording on the documentation could use some refactoring 😅 Hooks are not yet optimized to be tree-shakable. They're just designed to _only_ work in a non-production environment. 

Or perhaps, we could actually make those hooks dev-only 🤔

**Comment by dbuchet** [comment]
Yes we are using CRA, I'll try some workaround mentionned in your comment, thanks for your answer!
